src/mono/msbuild/AndroidAppBuilder/Templates/MainActivity.java

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+package net.dot;
+
+import android.app.AlertDialog;
+import android.app.Activity;
+import android.os.Bundle;
+
+public class MainActivity extends Activity
+{
+    @Override
+    protected void onCreate(Bundle savedInstanceState)
+    {
+        super.onCreate(savedInstanceState);
+
+        AlertDialog.Builder dlgAlert = new AlertDialog.Builder(this);
+        dlgAlert.setMessage("Use `adb shell am instrument -w ...` to run the tests");
```

Comment:
maybe change this to the full command `adb shell am instrument -w net.dot.HelloAndroid/net.dot.MonoRunner`.